eth2-ssv/cli/export_keys_from_mnemonic.go

```diff
@@ -0,0 +1,53 @@
+package cli
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/bloxapp/eth2-key-manager/core"
+	"github.com/spf13/cobra"
+	util "github.com/wealdtech/go-eth2-util"
+	"go.uber.org/zap"
+
+	"github.com/bloxapp/ssv/cli/flags"
+)
+
+// exportKeysCmd is the command to export private/public keys based on given mnemonic
+var exportKeysCmd = &cobra.Command{
+	Use:   "export-keys",
+	Short: "exports private/public keys based on given mnemonic",
+	Run: func(cmd *cobra.Command, args []string) {
+		mnemonicKey, err := flags.GetMnemonicFlagValue(cmd)
+		if err != nil {
+			Logger.Fatal("failed to get mnemonic key flag value", zap.Error(err))
+		}
+
+		index, err := flags.GetKeyIndexFlagValue(cmd)
+		if err != nil {
+			Logger.Fatal("failed to get key index flag value", zap.Error(err))
+		}
+
+		seed, err := core.SeedFromMnemonic(mnemonicKey, "")
```

Comment:
There should be validation on the mnemonic here, and ask as well for a 25th word passphrase